src/index.tsx

```
@@ -69,18 +44,50 @@ app.on(['POST', 'GET', 'OPTIONS'], '/graphql/*', async (context) => {
 
 app.all('/*', (context) => context.redirect(origin));
 
-serve({ fetch: app.fetch, port }, () =>
-  console.log(
-    `${styleText(['green', 'bold'], `${name}\n  ➜`)}  Server running on port ${styleText('bold', String(port))}.\n`,
-  ),
-);
-
-const setTitle = (title: string) => {
-  process.title = title;
-  if (process.stdout.isTTY) {
-    process.stdout.write(
-      `${String.fromCharCode(27)}]0;🚀 ${title}${String.fromCharCode(7)}`,
+if (process.env.npm_lifecycle_event === 'dev') {
+  try {
+    await prisma.$connect();
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(
+      `${styleText(['red', 'bold'], 'Prisma Database Connection Error')}\n`,
+      error,
     );
+    process.exit(1);
   }
```

Interesting! This is actually purely for prod since you want to fail fast when the db is not available in any other hosting scenario.

---

I see. Lemme move this out.